testcases/kernel/syscalls/ptrace/ptrace08.c

```diff
@@ -107,9 +88,13 @@ static void ptrace_try_kern_addr(unsigned long kern_addr)
 		(void *)offsetof(struct user, u_debugreg[0]),
 		(void *)kern_addr));
 
-	if (deffered_check) {
-		TEST(ptrace(PTRACE_POKEUSER, child_pid,
-			(void *)offsetof(struct user, u_debugreg[7]), (void *)1));
+	if (TST_RET == -1) {
+		addr = ptrace(PTRACE_PEEKUSER, child_pid,
+					  (void*)offsetof(struct user, u_debugreg[0]), NULL);
```

Comment:
NOTE: `(void*)` => `(void *)` (`make check-ptrace08` complains), but that can be fixed be us before merge.